jsm.go

```diff
@@ -100,15 +100,18 @@ type StreamConfig struct {
 	DenyDelete        bool            `json:"deny_delete,omitempty"`
 	DenyPurge         bool            `json:"deny_purge,omitempty"`
 	AllowRollup       bool            `json:"allow_rollup_hdrs,omitempty"`
+	AllowDirect       bool            `json:"allow_direct,omitempty"`
```

Comment:
I wonder whether exposing this option for a stream won't give the impression that we already support this feature in the client (and from what I see, we don't). Maybe it would be better if it goes together with actually sending direct requests + checking server compatibility?

Reply:
Possibly, but you can do the direct gets with nc.Request() with this enabled in the server..